akka-persistence/src/test/scala/akka/persistence/SnapshotFailureRobustnessSpec.scala

```diff
@@ -97,7 +97,12 @@ object SnapshotFailureRobustnessSpec {
 class SnapshotFailureRobustnessSpec extends PersistenceSpec(PersistenceSpec.config("leveldb", "SnapshotFailureRobustnessSpec", serialization = "off", extraConfig = Some(
   """
   akka.persistence.snapshot-store.local.class = "akka.persistence.SnapshotFailureRobustnessSpec$FailingLocalSnapshotStore"
-  akka.persistence.snapshot-store.local-delete-fail.class = "akka.persistence.SnapshotFailureRobustnessSpec$DeleteFailingLocalSnapshotStore"
+  akka.persistence.snapshot-store.local-delete-fail {
+    class = "akka.persistence.SnapshotFailureRobustnessSpec$DeleteFailingLocalSnapshotStore"
+    stream-dispatcher = "akka.persistence.dispatchers.default-stream-dispatcher"
+    dir = "target/snapshots-SnapshotFailureRobustnessSpec/"
+    max-load-attempts = 3
```

Comment:
new config path do not fallback with "akka.persistence.journal.leveldb". So user should set requirement parameters. Or should it fallback with defaults?